products/forms.py

```diff
@@ -1 +1,79 @@
 from django import forms
+from .models import Product
+from .models import Category
+from .models import Brand
```

Comment:
import in one line, if all the models are in one file. @Yshah112 @rinkishivani

Reply:
done